mirrord/config/src/env.rs

```diff
@@ -79,16 +78,15 @@ pub struct EnvFileConfig {
     ///
     /// For example, if the remote pod has an environment variable `REGION=1`, but this is an
     /// undesirable value, it's possible to use `overrides` to set `REGION=2` (locally) instead.
-    #[serde(rename = "override")]
-    pub overrides: Option<HashMap<String, String>>,
+    pub r#override: Option<HashMap<String, String>>,
```

Comment:
How about creating a `#[rename("override")]` attribute for the macro so we can avoid this?